cypress/support/elements/top-container.js

```diff
@@ -1,27 +1,27 @@
 class TopContainer {
-  getDrawContinents () {
-    return cy.get('[data-test=draw-continents')
-    // return cy.get('.interaction-selector > .large-button > .label').contains('Draw Continents');
+  getRefresh () {
+    return cy.get('[data-test=top-bar-refresh]')
   }
```

Comment:
Note that it could be a static method (my IDE complains about it):
`static getRefresh() { ... }`
Then, in the test file, you don't have to call constructor: `const top = new TopContainer(); (...) top.getRefresh()`. But you could simply say: `TopContainer.getRefresh()`. 

Not a big difference, a matter of taste, and I'm not expecting that you change it. I mention it, as my IDE highlights all these files. :wink: